src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -614,6 +829,33 @@ private GencodeFuncotation createUtrFuncotation(final VariantContext variant,
             }
         }
 
+        // Set GC Content:
+        gencodeFuncotationBuilder.setGcContent( calculateGcContent( reference, gcContentWindowSizeBases ) );
```

Comment:
Why is this in the funcotation factory class and not a datasource class?

Reply:
The `GencodeFuncotationBuilder` follows the builder pattern and procedurally assembles a `GencodeFuncotation` inside the `FuncotationFactory`.

Reply:
Got it.  I think I misunderstood what was happening here when I wrote the review comment.  No action required.